client/my-sites/site-monitoring/components/site-monitoring-tab-panel/index.tsx

```diff
@@ -12,29 +15,32 @@ export const tabs = [
 ];
 
 interface SiteMonitoringTabPanelProps {
-	children: React.ComponentProps< typeof TabPanel >[ 'children' ];
 	selectedTab?: SiteMonitoringTab;
 	className?: string;
-	onSelected?: ( tabName: SiteMonitoringTab ) => void;
 }
 
 export const SiteMonitoringTabPanel = ( {
-	children: renderContents,
 	selectedTab = 'metrics',
 	className,
-	onSelected,
 }: SiteMonitoringTabPanelProps ) => {
+	const siteSlug = useSelector( getSelectedSiteSlug ) as string;
 	return (
-		<TabPanel
-			initialTabName={ selectedTab }
+		<SectionNav
+			selectedText={ selectedTab }
 			className={ classnames( 'site-monitoring-tab-panel', className ) }
-			tabs={ tabs }
-			onSelect={ ( tabName ) => {
-				updatePageQueryParam( tabName as SiteMonitoringTab );
-				onSelected?.( tabName as SiteMonitoringTab );
-			} }
 		>
-			{ ( tab ) => renderContents( tab ) }
-		</TabPanel>
+			<NavTabs>
+				{ tabs.map( ( { name, title } ) => {
+					return (
+						<NavItem
+							key={ name }
+							path={ `/site-monitoring/${ siteSlug }/${ name }` }
```

Comment:
If you click on 'Metrics', can we keep the URL as `/site-monitoring/${ siteSlug }` instead of `/site-monitoring/${ siteSlug }/${ name }`